src/InteractiveBridge.ts

```diff
@@ -207,82 +226,183 @@ export class InteractiveBridge extends LiveryBridge {
-        );
-      }
-      return value;
-    }
-
     return this.sendCommand('subscribeFullscreen', undefined, (value) =>
-      listener(validate(value)),
-    ).then(validate);
+      listener(validateBoolean(value)),
+    ).then(validateBoolean);
+  }
+
+  /**
+   * Returns promise of current mode of playback, e.g. how to buffer, sync, adapt quality, manage stalls, etc.
+   * and calls back `listener` with any subsequent mode changes.
+   */
+  subscribeMode(listener: (mode: PlaybackMode) => void) {
+    return this.sendCommand('subscribeMode', undefined, (mode) =>
+      listener(playbackMode.parse(mode)),
+    ).then((mode) => playbackMode.parse(mode));
+  }
+
+  /**
+   * Returns promise of current LiveryPlayer muted state
+   * and calls back `listener` with any subsequent muted changes.
+   */
+  subscribeMuted(listener: (muted: boolean) => void) {
+    return this.sendCommand('subscribeMuted', undefined, (value) =>
+      listener(validateBoolean(value)),
+    ).then(validateBoolean);
   }
 
   /**
    * Returns promise of current LiveryPlayer window orientation (`'landscape' \| 'portrait'`)
    * and calls back `listener` with any subsequent orientations.
+   *
+   * @deprecated Will be removed in the next major version.
    */
-  subscribeOrientation(listener: (orientation: Orientation) => void) {
-    function validate(value: unknown) {
-      if (value !== 'landscape' && value !== 'portrait') {
-        const strValue = stringify(value);
-        throw new Error(
-          `subscribeOrientation value: ${strValue}, should be: "landscape" | "portrait"`,
-        );
-      }
-      return value;
-    }
-
+  subscribeOrientation(listener: (value: Orientation) => void) {
     return this.sendCommand('subscribeOrientation', undefined, (value) =>
-      listener(validate(value)),
-    ).then(validate);
+      listener(orientation.parse(value)),
+    ).then((value) => orientation.parse(value));
+  }
+
+  /**
+   * Returns promise of current playback paused state
+   * and calls back `listener` with any subsequent paused state updates.
+   */
+  async subscribePaused(listener: (value: boolean) => void) {
+    const state = await this.subscribePlaybackState((value) =>
+      listener(pausedState.safeParse(value).success),
+    );
+    return pausedState.safeParse(state).success;
+  }
+
+  /**
+   * Returns promise of current LiveryPlayer playback state
+   * and calls back `listener` with any subsequent state updates.
+   */
+  subscribePlaybackState(listener: (value: PlaybackState) => void) {
+    return this.sendCommand('subscribePlaybackState', undefined, (value) =>
+      listener(playbackState.parse(value)),
+    ).then((value) => playbackState.parse(value));
+  }
+
+  /**
+   * Returns promise of current playback playing state
+   * and calls back `listener` with any subsequent playing state updates.
+   */
+  async subscribePlaying(listener: (value: boolean) => void) {
```

Comment:
See subscribePaused above; please use`reducedSubscribe()` and a simple reducer function here.